index.html

```diff
@@ -705,31 +710,65 @@
                                 var inputPort, outputPort, controlPort, commandPort;
                                 if (cellvar.model.in.height == null) {
                                     inputPort = 0;
-                                } else {
+                                }
+                                else {
                                     inputPort = cellvar.model.in.height;
                                 }
                                 if (cellvar.model.out.height == null) {
                                     outputPort = 0;
-                                } else {
+                                }
+                                else {
                                     outputPort = cellvar.model.out.height;
                                 }
                                 if (cellvar.model.evtin.height == null) {
                                     controlPort = 0;
-                                } else {
+                                }
+                                else {
                                     controlPort = cellvar.model.evtin.height;
                                 }
                                 if (cellvar.model.evtout.height == null) {
                                     commandPort = 0;
-                                } else {
+                                }
+                                else {
                                     commandPort = cellvar.model.evtout.height;
                                 }
+
+                                var flip = false, mirror = false;
+                                if(cell.style.indexOf('stencilFlipV') == -1){
```

Comment:
Ya. I have made changes to styleToobject function. Check it in my latest commit.